source/auth/tests/mongodb-oidc.rst

```diff
@@ -141,6 +142,83 @@ Allowed Hosts Ignored
 - Assert that a ``find`` operation succeeds.
 - Close the client.
 
+Azure Automatic Auth
+==================
+
+Drivers MUST be able to authenticate using the "azure" provider workflow, using
+an Azure VM provisioned using the helper scripts in Drivers Evergreen Tools.
+These tests will most likely need to be run in a separate test file from the
+rest of the tests, to avoid needing to skip multiple tests.
```

Comment:
Maybe it's worth noting here that the URI for all tests is the same, and must be equal to:

`mongodb://localhost/?authMechanism=MONGODB-OIDC&authMechanismProperties=PROVIDER_NAME:azure,TOKEN_AUDIENCE:api://${AZUREOIDC_CLIENTID}`

Then each individual test can reuse the same URI. Thoughts? I did the same as Python and just set it as the `MONGODB_URI` and had all the clients use that.